DParser2/Resolver/DTypeToCodeVisitor.cs

```diff
@@ -98,7 +98,27 @@ public void VisitDelegateType(DelegateType t)
 		void VisitDSymbol(DSymbol t)
 		{
 			var def = t.Definition;
-			sb.Append(def != null ? def.ToString(false, true) : "<Node object no longer exists>");
+			if (def == null)
+				sb.Append("<Node object no longer exists>");
+			else
+			{
+				sb.Append(def.Name);
```

Comment:
Sollte hier vielleicht `GetNodePath verwenden? dmd zeigt den module name aber auch in stringof nicht an.

Reply:
Erstmal so lassen, ich denke, so geht das.